src/e2e-test/features/successfactors/RunTime.feature

```diff
@@ -0,0 +1,130 @@
+# Copyright © 2022 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+
+@SuccessFactorsSource
+@Smoke
+@Regression
+Feature: SuccessFactors Source - Run time scenarios
+
+  @BATCH-TS-SCFA-DSGN-RNTM-01 @BQ_SINK_TEST
+  Scenario: Verify user should be able to preview the pipeline when plugin is configured for a valid Entity name
+    When Open Datafusion Project to configure pipeline
+    And Select data pipeline type as: "Batch"
+    And Select plugin: "SAP SuccessFactors" from the plugins list as: "Source"
+    And Navigate to the properties page of plugin: "SAP SuccessFactors"
+    And Enter input plugin property: "referenceName" with value: "Referencename"
+    And Enter input plugin property: "baseURL" with value: "admin.baseurl"
+    And Enter input plugin property: "entityName" with value: "BadgeTemplates"
+    And Enter input plugin property: "username" with value: "admin.username"
+    And Enter input plugin property: "password" with value: "admin.password"
+    And Click on the Validate button
+    And Capture the generated Output Schema
+    And Close the Plugin Properties page
+    And Select Sink plugin: "BigQueryTable" from the plugins list
+    And Navigate to the properties page of plugin: "BigQuery"
+    And Configure BigQuery sink plugin for Dataset and Table
+    And Validate "BigQuery" plugin properties
+    And Close the Plugin Properties page
+    And Connect source as "SAP-SuccessFactors" and sink as "BigQueryTable" to establish connection
+    And Save the pipeline
+    And Preview and run the pipeline
+    And Verify the preview of pipeline is "success"
+    Then Verify sink plugin's Preview Data for Input Records table and the Input Schema matches the Output Schema of Source plugin
+
+  @BATCH-TS-SCFA-DSGN-RNTM-02 @BQ_SINK_TEST
+  Scenario: Verify user should be able to run and deploy the pipeline when plugin is configured for valid Entity name
```

Comment:
"should be able to deploy and run the pipeline"

Reply:
updated